target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490.dtsi

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz5490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 5490/5491";
```

Comment:
The commit message says it has a VDSL2 modem is this true?

Reply:
Technically yes, but I'm not sure it's possible to make use of it without a port on the back 🤔
https://boxmatrix.info/wiki/FRITZ!Box_5490

Reply:
Yes the digital parts are on the SoC, but they probably did not solder the analog frond end onto the board.